composer.json

```
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
-        "cakephp/console": "^5.0",
+        "cakephp/cakephp": "^5.0",
```

```suggestion
        "cakephp/console": "^5.1.5",
```

To be merged when 5.1.5 is released :slightly_smiling_face: